chore: Updated metrics alerting for bootnodes

PhilWindle opened the pull request:
This PR adds contact points and policies for routing alerts from 'network' labels.

spypsy commented on spartan/metrics/grafana/alerts/contactpoints.yaml:
```diff
@@ -257,3 +257,115 @@ contactPoints:
+      - uid: alertsmainnetbynetwork
+      - type: slack
+        settings:
+          url: $SLACK_WEBHOOK_TESTNET_URL
```
Think you meant mainnet URL here

PhilWindle committed:
chore: Updated metrics alerting for bootnodes

This PR adds contact points and policies for routing alerts from 'network' labels.

Co-authored-by: Phil Windle <philip.windle@gmail.com>
